### PR TITLE
issue:#3975912: Convert PDR to work with counters from the secondary telemetry instead of a dynamic instance

### PR DESCRIPTION
**Pull request by egershonNvidia**
## What
replacing the dynamic telemetry session with secondary telemetry.
Removing all the parts connected to the dynamic telemetry session.
we dont have telemetry collector class, this is another task.

## Why ?
[Task](https://redmine.mellanox.com/issues/3975912)
first task in order to merge the ALM plugin and PDR plugin, we want to take secondary telemetry in the code.

## How ?
It is optional, but for complex PRs please provide information about the design,
architecture, approach, etc.

## Testing ?
Changed the consts in the tests with the same constants to work as well.

## Special triggers
_Use the following phrases as comments to trigger different runs_

* ```bot:retest``` rerun Jenkins CI (to rerun GitHub CI, use "Checks" tab on PR page and rerun _all_ jobs)
* ```bot:upgrade``` run additional update tests

### CODE REVIEW

File: plugins/pdr_deterministic_plugin/ufm_sim_web_service/isolation_mgr.py [hunk 1]
```diff
@@ -456,13 +439,11 @@ def check_temp_issue(self, port_obj, row, timestamp):
         if not self.temp_check:
             return None
         cable_temp = get_counter(Constants.TEMP_COUNTER, row, default=None)
-        if cable_temp is not None and not pd.isna(cable_temp):
-            if cable_temp in ["NA", "N/A", "", "0C", "0"]:
+        if cable_temp is not None and not numpy.isnan(cable_temp):
+            if cable_temp in ["NA", "N/A", "", "0C"]:
```
Comment (vg12345) [inline plugins/pdr_deterministic_plugin/ufm_sim_web_service/isolation_mgr.py:443]: "0" option was add as part of this PR. If I'm not wrong, it was reproduced on some testing environment.
https://github.com/Mellanox/ufm_sdk_3.0/pull/193

Comment (TalLerner) [reply plugins/pdr_deterministic_plugin/ufm_sim_web_service/isolation_mgr.py]: @egershonNvidia can you please add the "0" back to the string array?

**Comment by TalLerner** [comment]
Please fix the Git Action CI

**Review by TalLerner — APPROVED** [review]

**Comment by egershonNvidia** [comment]
bot:retest

File: plugins/pdr_deterministic_plugin/ufm_sim_web_service/collector.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import pandas as pd
```
Comment (drorlevy) [inline plugins/pdr_deterministic_plugin/ufm_sim_web_service/collector.py:1]: missing copyrights (maybe that's why PR fails CI)

File: plugins/pdr_deterministic_plugin/ufm_sim_web_service/collector.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+import urllib.error
+import logging
+from constants import PDRConstants
+class Collector:
```
Comment (drorlevy) [inline plugins/pdr_deterministic_plugin/ufm_sim_web_service/collector.py:5]: pylint: missinh empty lines

Comment (drorlevy) [inline plugins/pdr_deterministic_plugin/ufm_sim_web_service/collector.py:5]: change to "TelemetryCollector"? other collected data comes from ufm_communication_mgr

**Comment by egershonNvidia** [comment]
bot:retest

File: plugins/pdr_deterministic_plugin/ufm_sim_web_service/collector.py (hunk 2 above)
Comment (egershonNvidia) [reply plugins/pdr_deterministic_plugin/ufm_sim_web_service/collector.py:1]: no no it something related to the connection from the runner to the machine, you can see the git action works.

**Review by drorlevy — APPROVED** [review]

**Pull request merged**